.changeset/config.json

```diff
@@ -0,0 +1,24 @@
+{
+    "$schema": "https://unpkg.com/@changesets/config@2.3.1/schema.json",
+    "changelog": [
+        "./es-changelog-config.js",
+        { "repo": "EventStore/Design-System" }
+    ],
+    "commit": false,
+    "fixed": [],
+    "linked": [],
+    "access": "public",
+    "baseBranch": "main",
+    "updateInternalDependencies": "patch",
+    "ignore": [
+        "@eventstore-ui/docs",
+        "stores-demo",
+        "router-demo",
+        "@eventstore-ui/docs",
```

Comment:
Double docs. 

We also need to add:
[preview-builder](https://github.com/EventStore/Design-System/blob/main/tools/preview-builder/package.json)